app/service/rest_svc.py

```diff
@@ -176,8 +176,15 @@ async def list_payloads(self):
         payload_dirs = [pathlib.Path.cwd() / 'data' / 'payloads']
         payload_dirs.extend(pathlib.Path.cwd() / 'plugins' / plugin.name / 'payloads'
                             for plugin in await self.get_service('data_svc').locate('plugins') if plugin.enabled)
-        return set(p.name for p_dir in payload_dirs for p in p_dir.glob('*')
-                   if p.is_file() and not p.name.startswith('.'))
+        payloads = set()
+        for p_dir in payload_dirs:
+            for p in p_dir.glob('*'):
+                if p.is_file() and not p.name.startswith('.'):
+                    if p.name.endswith('.xored'):
+                        payloads.add(p.name.replace('.xored', ''))
```

Comment:
can we address this in a way that doesn't hard code a file extension into the codebase?

Reply:
It's hardcoded a couple other places too.  Switching it might be tricky and affect how we store encoded payloads and declare them in ability files.  Not saying it wouldn't be a good change to make, but this PR addresses an annoying bug and I'm unsure if we want to hold the fix up. 

We could probably easily push this logic into a `file_svc` function which might be a little nicer though.